client/state/jetpack-connect/actions.js

```diff
@@ -336,7 +336,7 @@ export default {
 					type: JETPACK_CONNECT_AUTHORIZE_RECEIVE,
 					siteId: client_id,
 					data: null,
-					error: error
+					error: pick( error, [ 'error', 'status', 'message' ] )
```

Comment:
Sure! This is necessary to make the error more testable, because if an error is thrown, it contains a lot of information, including the error stack trace. With this change we only pull whatever we might be needing from the error, which also makes it easier to test.